aea/cli/register.py

```diff
@@ -49,10 +56,16 @@ def do_register(
     :param email: str email.
     :param password: str password.
     :param password_confirmation: str password confirmation.
+    :param subscribe: bool flag for developers subscription on register.
 
     :return: None
     """
     username = validate_author_name(username)
     token = register_new_account(username, email, password, password_confirmation)
     update_cli_config({AUTH_TOKEN_KEY: token})
+    if subscribe or click.confirm("Do you want to subscribe for developer news?"):
+        click.echo(
+            "Please visit `https://aea-registry.fetch.ai/mailing-list` "
+            "to subscribe for developer news"
+        )
```

Comment:
so the only effect of `--subscribe` is to print the URL to the command line?

Reply:
Yes. That was the requirement.